spec/index.bs

```diff
@@ -3063,7 +3063,7 @@ enum GPUCompareFunction {
             1. Let |s| be a new {{GPUSampler}} object.
             1. Set |s|.{{GPUSampler/[[descriptor]]}} to |descriptor|.
             1. Set |s|.{{GPUSampler/[[isComparison]]}} to `false` if the {{GPUSamplerDescriptor/compare}} attribute
-                    of |s|.{{GPUSampler/[[descriptor]]}} is `null` or undefined. Otherwise, set it to `true`.
+                    of |s|.{{GPUSampler/[[descriptor]]}} does not [=map/exist=] or is `null`. Otherwise, set it to `true`.
```

Comment:
```suggestion
                    of |s|.{{GPUSampler/[[descriptor]]}} does not [=map/exist=]. Otherwise, set it to `true`.
```

GPUSamplerDescriptor.compare cannot actually be null.